### ISSUE TITLE
tridactyl-1.14.1: primary search always defaults to google

### ISSUE THREAD

**Issue by m1027** [tridactyl/tridactyl#1142]
Since tridactyl-1.14.1 tridactyl does not respect the primary search engine anymore that has been set up in firefox options.

To reproduce:

(1) Remove all search engines in firefox options except for DuckDuckGo (not Google).
(2) Search something in Tridactyl like usual (enter :o and then any term to search for).

Result: Tridactyl opens google search.
Expected result: Searching via DuckDuckGo.

Workaround: Explicitly search like so: :o duckduckgo <search terms>
This is a bit annoying... Thanks!

**Comment by glacambre** [comment]
Hi, thanks for opening this issue :)

This is a problem we're aware of (https://github.com/tridactyl/tridactyl/issues/333) and plan on fixing (https://github.com/tridactyl/tridactyl/issues/792). Unfortunately as you'll see in #792, it is blocked by a few other issues and will take some time before being fixed.

In the meantime, you can manually set the searchengine firefox should use by using `:set searchengine duckduckgo`, as mentioned in the [FAQ](https://github.com/tridactyl/tridactyl#frequently-asked-questions).

I'm going to close this issue for now because it's a duplicate of #333. Feel free to reopen it/let me know if I misunderstood your problem.
